### PR TITLE
update the z-index of .app-home-body-content class

### LINKED ISSUE #568: 🐛 [Bug report]:  Content overlap on sidebar due to same z-index value
**Describe the bug**

The header component and  the section (with class name - "app-home-body") that has div (with class name - "app-home-body-content") has same z-index value (i.e 10) which causes the content within them to overlap during scroll when the sidebar is open.
The overlap of content makes it unable to view the links (and access them) that are present in the sidebar.

**To Reproduce**
1. Go to the home page of reactplay.io
2. Click on the menu button while at the top of the page
3. Scroll the screen while the sidebar is open
4. See the content of section app-home-body overlapping with the sidebar

**Expected behavior**

The content of  sidebar and div (with class name - "app-home-body-content") should not overlap so that the contents within them are clearly visible and links are properly clickable 

**Screenshots**

issue_img1

issue_img2

**Desktop (please complete the following information):**
All devices , any browser
 - 
**Smartphone (please complete the following information):**
All devices , any browser

**Additional context**
I would really like to work on this issue as it can be my first contribution to a public git hub repository

### PR DESCRIPTION
**Pull request by yash91989201**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Changed the z-index value of .app-home-body-content class to 5 so that it doesnot clash with sidebar z-index value and causes a 
content overlap. I changed the value to 5 but not 1 because of an image that is present along with .app-home-body-content class as its sibling
Fixes #568 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by atapas** [comment]
@yash91989201 I have marked the PR for Hacktoberfest.. Make sure you register for the event... Follow the Forum discussion on our discord.

Coming back the PR, the Tests are failing, please take a look.

**Comment by yash91989201** [comment]
Sure I'll look into it

**Comment by atapas** [comment]
> Sure I'll look into it

Test Passed now.

**Comment by yash91989201** [comment]
Please wait until october 1 for merging this pr, because then only it will count under my hacktoberfest contribution

**Comment by atapas** [comment]
Yes sir, in my radar 😀

**Comment by Sachin-chaurasiya** [comment]
@yash91989201, Can you fix the formatting as the entire CSS file content is getting changed?

File: src/common/home/home.css
Comment (Sachin-chaurasiya) [inline src/common/home/home.css:28]: @yash91989201 , Just to confirm, this is the only change that you want to make right?

Comment (yash91989201) [reply src/common/home/home.css:28]: Yes, the index value was only causing the content overlap

Comment (Sachin-chaurasiya) [reply src/common/home/home.css:28]: Can you fix the formatting as the entire CSS file content is getting changed?

Comment (yash91989201) [reply src/common/home/home.css:28]: Could you please suggest me how to do it?

Comment (yash91989201) [reply src/common/home/home.css:28]: > Can you fix the formatting as the entire CSS file content is getting changed?

How is the file content getting changed??

Comment (Sachin-chaurasiya) [reply src/common/home/home.css:28]: @yash91989201 , You can disable the formatting, so it will not format the CSS content.
 You can use this extension https://marketplace.visualstudio.com/items?itemName=tombonnike.vscode-status-bar-format-toggle.

**Comment by yash91989201** [comment]
done I, turned the formatting off and then I changed the z-index value for the final time and I commit the changes

**Comment by Sachin-chaurasiya** [comment]
> Done, I turned the formatting off and then I changed the z-index value for the final time and I commit the changes. Are there any errors now ??

I will check and let you know.

**Comment by Sachin-chaurasiya** [comment]
@yash91989201 , Still there is a formatting issue.

image

**Comment by yash91989201** [comment]
Okay, I'll fit the issue this time

**Comment by yash91989201** [comment]
Everything fixed , now there will be no problem in merging (i hope so :) ) .

**Comment by atapas** [comment]
Hi @Sachin-chaurasiya 

Could you please approve this PR if all is done from your end?

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by atapas — APPROVED** [review]
The changes are looking good to me. Accepting.

**Comment by atapas** [comment]
@yash91989201 Thanks for your contribution. The next step is socialising your contribution and getting started with the next issue! Keep contributing.

**Comment by atapas** [comment]
@all-contributors please add @yash91989201  for Code

**Pull request merged**
